<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>UserPermissionForm</class>
    <widget class="QWidget" name="UserPermissionForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>767</width>
                <height>420</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <widget class="QLabel" name="labelTitleAllocatePermissions">
            <property name="geometry">
                <rect>
                    <x>620</x>
                    <y>10</y>
                    <width>141</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                </font>
            </property>
            <property name="text">
                <string>用户管理-权限分配</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
                <rect>
                    <x>240</x>
                    <y>70</y>
                    <width>261</width>
                    <height>24</height>
                </rect>
            </property>
            <layout class="QFormLayout" name="formLayout_5">
                <item row="0" column="0">
                    <widget class="QLabel" name="labelTabAllocatePermissionsUserId">
                        <property name="text">
                            <string>目标用户的ID：</string>
                        </property>
                        <property name="alignment">
                            <set>Qt::AlignmentFlag::AlignCenter</set>
                        </property>
                    </widget>
                </item>
                <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEditTabAllocatePermissionsUserId">
                        <property name="maxLength">
                            <number>6</number>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QGroupBox" name="groupBoxTabAllocatePermissionsResetPwd">
            <property name="geometry">
                <rect>
                    <x>430</x>
                    <y>140</y>
                    <width>241</width>
                    <height>191</height>
                </rect>
            </property>
            <property name="title">
                <string>重置用户密码</string>
            </property>
            <widget class="QPushButton" name="pBtnAllocateAllocatePermissionsResetPwd">
                <property name="geometry">
                    <rect>
                        <x>70</x>
                        <y>120</y>
                        <width>141</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>确认重置</string>
                </property>
            </widget>
            <widget class="QWidget" name="formLayoutWidget_6">
                <property name="geometry">
                    <rect>
                        <x>30</x>
                        <y>50</y>
                        <width>181</width>
                        <height>50</height>
                    </rect>
                </property>
                <layout class="QFormLayout" name="formLayout_6">
                    <item row="0" column="0">
                        <widget class="QLabel" name="labelTabAllocatePermissionsResetPwd1">
                            <property name="text">
                                <string>新密码</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="lineEditTabAllocatePermissionsResetPwd1">
                            <property name="maxLength">
                                <number>12</number>
                            </property>
                            <property name="echoMode">
                                <enum>QLineEdit::EchoMode::Password</enum>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="labelTabAllocatePermissionsResetPwd2">
                            <property name="text">
                                <string>再输一次</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="lineEditTabAllocatePermissionsResetPwd2">
                            <property name="maxLength">
                                <number>12</number>
                            </property>
                            <property name="echoMode">
                                <enum>QLineEdit::EchoMode::Password</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QGroupBox" name="groupBoxTabAllocatePermissionsChUserType">
            <property name="geometry">
                <rect>
                    <x>129</x>
                    <y>140</y>
                    <width>241</width>
                    <height>191</height>
                </rect>
            </property>
            <property name="title">
                <string>变更用户身份</string>
            </property>
            <widget class="QComboBox" name="comboBoxTabAllocatePermissionsUserType">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>60</y>
                        <width>161</width>
                        <height>22</height>
                    </rect>
                </property>
                <item>
                    <property name="text">
                        <string>OB-普通读者</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>FB-教职员读者</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>LA-图书管理员</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>UA-用户管理员</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>SA-系统管理员</string>
                    </property>
                </item>
                <item>
                    <property name="text">
                        <string>DS-冻结状态</string>
                    </property>
                </item>
            </widget>
            <widget class="QPushButton" name="pBtnAllocateAllocatePermissionsUserType">
                <property name="geometry">
                    <rect>
                        <x>40</x>
                        <y>110</y>
                        <width>161</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="text">
                    <string>确定变更</string>
                </property>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
